sdks/java/io/elasticsearch/src/main/java/org/apache/beam/sdk/io/elasticsearch/ElasticsearchIO.java

```diff
@@ -148,31 +145,31 @@ private ElasticsearchIO() {}
 
   private static final ObjectMapper mapper = new ObjectMapper();
 
-  static JsonNode parseResponse(Response response) throws IOException {
+  public static JsonNode parseResponse(Response response) throws IOException {
```

Comment:
Well, I forgot to put back to private / package local many of the visibilities that were set to public because of the multi-module architecture, thanks. There is still a question: must the test modules use a per es-version  package name or must they use `org.apache.beam.sdk.io.elasticsearch`.   If the former, then some visibilities must remain public.

Reply:
I think it's fine to use the same package name. Generally fine to keep some things public if necessary, but please minimize the set of such things, and mark them `@VisibleForTesting` if they're not intended to actually be used by users.

Reply:
done: I narrowed down to the maximum all the visibilities